src/main/java/com/ericsson/gerrit/plugins/eiffel/handlers/DataBaseHandler.java

```diff
@@ -0,0 +1,233 @@
+        String eventID = "";
+
+        String sqlSelectStatement = String.format("SELECT * FROM %s WHERE %s=?", table, table.keyName);
+        try (Connection connection = connect();
+                PreparedStatement preparedStatement = connection.prepareStatement(sqlSelectStatement)) {
+
+            preparedStatement.setString(1, keyValue);
+            eventID = executeQuery(preparedStatement);
+
+        } catch (SQLException e) {
+            LOGGER.error("Error when trying to fetch values from database: {}\n{}", e.getMessage(), e);
+        }
+
+        return eventID;
+    }
+
+    /**
+     * This function updates value to the given table. The keyValue value is different depending on
+     * Table (branch name for scm and change-id for scmp)
+     *
+     * @param table
+     * @param keyValue
+     * @param eiffelevent
+     * @throws ConnectException
+     * @throws SQLException
+     */
+    public void updateInto(final Table table, final String keyValue, final String eiffelevent)
+            throws ConnectException, SQLException {
+        String sqlUpdateStatement = String.format("UPDATE %s SET %s=? WHERE %s=?", table, EVENT_ID_KEY, table.keyName);
+        executeUpdate(sqlUpdateStatement, keyValue, eiffelevent);
+    }
+
+    /**
+     * This function inserts values to the given table. The keyValue value is different depending on
+     * Table (branch name for scm and change-id for scmp)
+     *
+     * @param table
+     * @param keyValue
+     * @param eiffelevent
+     * @throws SQLException
+     * @throws ConnectException
+     */
+    public void insertInto(final Table table, final String keyValue, final String eiffelevent)
+            throws SQLException, ConnectException {
+        String sqlInsertStatement = String.format("INSERT INTO %s(%s,%s) VALUES(?,?)", table, EVENT_ID_KEY,
+                table.keyName);
+        executeUpdate(sqlInsertStatement, keyValue, eiffelevent);
+
+    }
+
+    /**
+     * This function will create a database structure in the database file if it does not exist.
+     *
+     * @throws ConnectException
+     */
+    private void createNewDatabase() throws ConnectException {
+        try (Connection connection = connect()) {
+            if (connection != null) {
+                connection.getMetaData();
+                connection.close();
+            }
+
+        } catch (SQLException e) {
+            LOGGER.error("Error when trying to create new database: {}\n{}", e.getMessage(), e);
+        }
+    }
+
+    /**
+     * This function loops the values in the Table enum and executes the create table command for each
+     * table and creates it in the database if it not already exist.
+     *
+     * @throws ConnectException
+     */
+    private void createTables() throws ConnectException {
+        try (Connection connection = connect(); Statement statement = connection.createStatement()) {
```

Comment:
Add this line to a method maybe? It's a big one liner

Reply:
done